Tests/LibGfx/TestImageDecoder.cpp

```diff
@@ -489,6 +489,18 @@ TEST_CASE(test_tiff_rgb_alpha)
     EXPECT_EQ(frame.image->get_pixel(60, 75), Gfx::Color::NamedColor::Red);
 }
 
+TEST_CASE(test_tiff_palette_alpha)
+{
+    auto file = MUST(Core::MappedFile::map(TEST_INPUT("tiff/rgb_palette_alpha.tiff"sv)));
```

Comment:
can you say how you created this file? palette + alpha outside of palette is a bit whacky – can gimp open these?